packages/ag-charts-enterprise/benchmarks/areaStacked.test.ts

```diff
@@ -5,7 +5,7 @@ import { AgCartesianChartOptions } from 'ag-charts-types';
 import { benchmark, isAtOrAfterVersion, setupBenchmark } from './benchmark';
 
 let describe = originalDescribe;
-if (!isAtOrAfterVersion(12, 0, 0)) {
+if (!isAtOrAfterVersion(12, 1, 0)) {
```

Comment:
Make this `12, 2, 0` if this test wouldn't have worked before the current release cycle?